includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -69,12 +69,10 @@
 use WCPay\Payment_Methods\Becs_Payment_Method;
 use WCPay\Payment_Methods\CC_Payment_Method;
 use WCPay\Payment_Methods\Eps_Payment_Method;
-use WCPay\Payment_Methods\Giropay_Payment_Method;
```

Comment:
Giropay has been deprecated before, so just taking the liberty of removing redundant imports and tests for Giropay as well, wherever I encountered them.